tox.ini

```diff
@@ -9,11 +9,23 @@ passenv = CI TRAVIS_BUILD_ID TRAVIS TRAVIS_BRANCH TRAVIS_JOB_NUMBER TRAVIS_PULL_
 
 [testenv:py27]
 commands =
+  py.test -n 8 --color=yes --timeout 60 []
+
+[testenv:py35]
+# remove bash & pipe & grep hack after cryptography ships with openssl 1.1.0
+whitelist_externals = bash
+commands =
+  bash -c 'set -o pipefail ; py.test -n 8 --color=yes --timeout 60 test/netlib test/mitmproxy/script test/pathod/test_utils.py test/pathod/test_log.py test/pathod/test_language_generators.py test/pathod/test_language_writer.py test/pathod/test_language_base.py test/pathod/test_language_http.py test/pathod/test_language_websocket.py test/pathod/test_language_http2.py 2>&1 | grep -v Cryptography_locking_cb'
+
+[testenv:py27-ci]
+basepython = python2.7
+commands =
   py.test --cov netlib --cov mitmproxy --cov pathod --color=yes --timeout 60 ./test
   codecov -e TOXENV
 
-[testenv:py35]
+[testenv:py35-ci]
 # remove bash & pipe & grep hack after cryptography ships with openssl 1.1.0
+basepython = python3.5
```

Comment:
Why did you set `basepython` here? Isn't this auto-detected (at least on my system)?